client/state/plans/actions.js

```diff
@@ -81,7 +81,7 @@ export const requestPlans = () => {
 
 		return wpcom
 			.plans()
-			.list()
+			.list( { apiVersion: '1.1' } )
```

Comment:
Actually I forgot change the version here. Talking about this @daniloercoli told us that they are using `1.2` for mobile app.